starkinfra/staticbrcode/__staticbrcode.py

```diff
@@ -0,0 +1,115 @@
+
+
+_resource = {"class": StaticBrcode, "name": "StaticBrcode"}
+
+
+def create(brcodes, user=None):
+    """# Create StaticBrcodes
+    Send a list of StaticBrcode objects for creation at the Stark Infra API
+    ## Parameters (required):
+    - StaticBrcode [list of StaticBrcode objects]: StaticBrcode objects to be created in the API.
+    ## Parameters (optional):
+    - user [Organization/Project object, default None]: Organization or Project object. Not necessary if starkinfra.user was set before function call
+    ## Return:
+    - list of StaticBrcode objects with updated attributes
+    """
+    return rest.post_multi(resource=_resource, entities=brcodes, user=user)
+
+
+def get(uuid, user=None):
+    """# Retrieve a specific StaticBrcode
+    Receive a single StaticBrcode object previously created in the Stark Infra API by its uuid
+    ## Parameters (required):
+    - uuid [string]: object unique uuid. ex: "97756273400d42ce9086404fe10ea0d6"
+    ## Parameters (optional):
+    - user [Organization/Project object, default None]: Organization or Project object. Not necessary if starkinfra.user was set before function call
+    ## Return:
+    - StaticBrcode object with updated attributes
+    """
+    return rest.get_id(resource=_resource, id=uuid, user=user)
+
+
+def query(limit=None, after=None, before=None, uuids=None, user=None):
+    """# Retrieve StaticBrcodes
+    Receive a generator of StaticBrcode objects previously created in the Stark Infra API
+    ## Parameters (optional):
+    - limit [integer, default None]: maximum number of objects to be retrieved. Unlimited if None. ex: 35
+    - after [datetime.date or string, default None] date filter for objects created only after specified date. ex: datetime.date(2020, 3, 10)
+    - before [datetime.date or string, default None] date filter for objects created only before specified date. ex: datetime.date(2020, 3, 10)
+    - uuids [list of strings, default None]: list of uuids to filter retrieved objects. ex: ["97756273400d42ce9086404fe10ea0d6", "e3da0b6d56fa4045b9b295b2be82436e"]
+    - user [Organization/Project object, default None]: Organization or Project object. Not necessary if starkinfra.user was set before function call
+    ## Return:
+    - generator of StaticBrcode objects with updated attributes
+    """
+    return rest.get_stream(
+        resource=_resource,
+        after=check_date(after),
+        before=check_date(before),
+        uuids=uuids,
+        limit=limit,
+        user=user,
+    )
+
+
+def page(cursor=None, limit=None, after=None, before=None, uuids=None, user=None):
+    """# Retrieve StaticBrcodes
+    Receive a list of StaticBrcode objects previously created in the Stark Infra API and the cursor to the next page.
+    ## Parameters (optional):
+    - cursor [string, default None]: cursor returned on the previous page function call
+    - limit [integer, default 100]: maximum number of objects to be retrieved. Unlimited if None. ex: 35
```

Comment:
Aqui mudar tmb para:
`- limit [integer, default 100]: maximum number of objects to be retrieved. Max = 100. ex: 35`